App.js

```diff
@@ -1,19 +1,17 @@
 import React, { useState, useEffect } from 'react';
 import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
-import LogoSvg from "./LogoSvg";
-import Quotes from "./Quotes";
```

Comment:
These components are now being imported and returned within their respective parent components.

Reply:
remove, clear from code change